CHANGELOG

```diff
@@ -1,4 +1,20 @@
 # Changelog
+## [1.0.2] - 2021-02-18
+### New features
+ - Support multiconnect profiles. (2c60c834)
+ - Support bond vlan+srcmac tx hashing option. (ebf5777)
+ - Support routes created by iproute command. (f39f763)
+ - New property `Inetrface.COPY_FROM_MAC` on bridge and bond interfaces. (b089323)
+ - Support IEEE 802.1X support. (f9a1da3)
+
+### Bug fixes
+ - Fix removing OVS `external_ids` settings. (57e2547)
+ - Fix modification of OVS `external_ids` settings. (c99e708)
+ - Fix removing IPv6 default gateway. (fd21172)
```

Comment:
- Improved the reliability on activating large amount of interfaces. (cbe5b57)

Reply:
Done. Thanks!